axis_backend/apps/clients/services/industry_service.py

```
@@ -0,0 +1,299 @@
+
+    @transaction.atomic
+    def update_industry(
+        self,
+        industry_id: str,
+        name: Optional[str] = None,
+        code: Optional[str] = None,
+        description: Optional[str] = None,
+        parent_id: Optional[str] = None,
+        external_id: Optional[str] = None,
+        metadata: Optional[Dict[str, Any]] = None,
+        **kwargs
```

Ruff (0.14.5)

Missing type annotation for `**kwargs`

(ANN003)